mobsf/StaticAnalyzer/views/ios/adnetworkidentifiers.py

```diff
@@ -0,0 +1,56 @@
+known_adnetworks = {
```

Comment:
What is the source of this list?

Reply:
There are organisations collecting these identifiers:

https://www.skanids.com/
https://github.com/skadnetwork/ad-network-ids